.github/workflows/bump-image-tags.yaml

```diff
@@ -0,0 +1,52 @@
+name: Bump Image Tags
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1" # Run at 00:00 UTC every Monday
+
+env:
+  team_reviewers: 2i2c-org/tech-team
+
+jobs:
+  bump-image-tags:
+    runs-on: ubuntu-latest
+    environment: github-app-token-generator
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # For each new config_path to monitor, add a new item in this matrix.
+          # The Action can read multiple paths to images, but not multiple config files.
+          - name: "pangeo-hubs common singleuser image"
+            config_path: "config/clusters/pangeo-hubs/common.values.yaml"
+            values_paths: ".basehub.jupyterhub.singleuser.image"
+          - name: "m2lines common singleuser/PyTorch/Tensorflow"
+            config_path: "config/clusters/m2lines/common.values.yaml"
+            values_paths: ".basehub.jupyterhub.singleuser.image .basehub.jupyterhub.singleuser.profileList[4].kubespawner_override.image .basehub.jupyterhub.singleuser.profileList[5].kubespawner_override.image"
```

Comment:
Can you add a comment to the values files here that if the ordering changes, we should also update the files here?

Separately, a feature request for the upstream action is more native support for profile_list, where it can then read image_override and do the bumps accordingly :)

Reply:
So we can do `...profileList[*]` which will return everything under the profileList key, but then we would need Python code to loop through each item and check that the kubespawner_override key is present and that in turn has the image key present. It's doable; I just don't know which is better: Relying on "magic" that the code will just "figure it out" or providing explicit/specific input? If we made this change and something went wrong with the profileList bumping, then the user would have to debug both their JMESPath query _and_ the source code. Whereas currently the only mistake that can be made right now is the user provides an incorrect JMESPath query.

Reply:
> 'anything with an image key under profileList'

I'll actually dig into this a little bit more